mods/default/README.txt

```diff
@@ -254,6 +250,10 @@ Topywo (CC BY-SA 3.0)
 Extex101 (CC BY-SA 3.0)
   default_large_cactus_seedling.png
 
+An0n3m0us (CC BY-SA 3.0):
+  heart.png, derived from texture by KevDoy (CC BY-SA 3.0)
+  bubble.png, derived from texture by BlockMen (CC BY-SA 3.0)
```

Comment:
There is room in these 2 lines for '--' and an 'a', as in similar lines above. Please could you format it like this:
`-- Derived from a texture by BlockMen (CC BY-SA 3.0)`
Currently it reads poorly.

Reply:
Done.